client/src/components/Navbar/Navbar.js

```diff
@@ -1,21 +1,114 @@
+      // The link end points for each menu item.
+      menuLinks: ["/", "/", "/", "/"],
+      // Each item in this array is a set for the dropdown on each menu link.
+      // This is an example for one drop menu item: { text: 'Explore People', to: '/' }
+      menuDropDown: [[], [], [], []]
+    };
+  }
 
-  // Default Class to apply to Component
-  let classList = `Navbar`;
+  // Handles activating the hamburger animation and displays the menu.
+  handleCondensedMenuClick = () => {
+    this.setState({ condensedMenuActive: !this.state.condensedMenuActive });
+    if (this.state.condensedMenuActive === false) {
+      this.setState({ condensedMenu: "Navbar__show" });
+    } else {
+      this.setState({ condensedMenu: "" });
+    }
+  }
 
+  // Sets the location of the triangle on the menu.
+  setPageMarker = () => {
+    let setPage = [];
+    for (let i = 0; i < 4; i++) {
+      if (i === this.props.page) {
+        setPage.push("Navbar__show");
+      } else {
+        setPage.push("");
+      }
+    }
+    this.setState({ pageMark: setPage });
+  }
 
-  return (
-    <div className={classList}>
-      <div className="navbar-logo">
-        <Logo clickable to="/route" />
+  // Creates the menu. If a 1 is passed into the function, drop down menus are created.
+  // Once link end points are known, they can be put into state and put in via this function
+  setMenu = (type) => {
+    if (type === 1) {
+      return this.state.menuItems.map((item, key) => (
+        <div className="Navbar__link-item">
+          <NavLink text={item} to={this.state.menuLinks[key]} menuList={this.state.menuDropDown[key]} />
+          <div className={`Navbar__triangle ${this.state.pageMark[key]}`}></div>
+        </div>
+      ));
+    } else {
+      return this.state.menuItems.map((item, key) => (
+        <div className="Navbar__link-item">
+          <NavLink text={item} to="/" menuList={[]} />
+        </div>
+      ));
+    }
+  }
+
+  clickHandler = () => {
+    console.log("Clicked!")
+  }
+
+  componentWillMount() {
+    this.setPageMarker();
+  }
+
+  render() {
+    return (
+      <div className="Navbar">
+        <div className="Navbar__logo">
+          <Logo clickable to="/route" />
+        </div>
+        <div className="Navbar__main">
+          <div className="Navbar__menu">{this.setMenu(1)}</div>
+          <div className="Navbar__right">
+            <Icon icon={"search"} size="2x" onClick={this.clickHandler} />
+            <Icon icon={["far", "comment-dots"]} size="2x" onClick={this.clickHandler} />
+            <ProfilePicture type="extra-small" link="#" image={this.props.profileImage} />
+          </div>
+        </div>
+        <div className="Navbar__burger">
+          <Burger onClick={this.handleCondensedMenuClick} active={this.state.condensedMenuActive} />
+          <div className={`Navbar__condensed-menu ${this.state.condensedMenu}`}>
+            {this.setMenu()}
+          </div>
+        </div>
       </div>
```

Comment:
Can you make it so `Navbar__main` is responsive, rather than having two version of the menu (`Navbar__main` and `Navbar__burger`)?

`Navbar__main` should look like the mock on laptop and desktop and use the hamburger menu and open and close on mobile and tablets.

Let's block off some time today for me to walk you through what I mean

Reply:
I changed it so that the burger menu only shows on mobile. I want talk to you on the best way to get the text areas of the menu to be responsive. I have some ideas, though.